riscv/src/continuations/bootloader64.rs

```diff
@@ -0,0 +1,649 @@
+use crate::code_gen::Register;
+
+use crate::continuations::bootloader::{
+    BOOTLOADER_INPUTS_PER_PAGE, BYTES_PER_WORD, DEFAULT_PC, MEMORY_HASH_START_INDEX,
+    MERKLE_TREE_DEPTH, NUM_PAGES_INDEX, N_LEAVES_LOG, PAGE_INPUTS_OFFSET, PAGE_NUMBER_MASK,
+    PAGE_SIZE_BYTES, PC_INDEX, REGISTER_NAMES, SHUTDOWN_START, WORDS_PER_HASH, WORDS_PER_PAGE,
+};
+
```

Comment:
This "new" file contains everything "removed" from file above `bootloader.rs` that is specific to the 64 bits version of the constraints, unmodified.